nextcord/flags.py

```diff
@@ -217,25 +217,25 @@ def _set_flag(self, o: int, toggle: bool) -> None:
             raise TypeError("Value to set for SystemChannelFlags must be a bool.")
 
     @flag_value
-    def join_notifications(self):
+    def join_notifications(self) -> int:
```

Comment:
Aren't these technically wrong as ``@flag_value`` modifies them to return bools

Reply:
Pyright recognises that.
![image](https://user-images.githubusercontent.com/34910574/206187165-0541ddc9-3291-4ace-93f6-7e62f6fd5315.png)

Reply:
Interesting, feel free to resolve then ig